docs/core/diagnostics/dotnet-gcdump.md

```diff
@@ -0,0 +1,110 @@
+---
+title: dotnet-gcdump - .NET Core
+description: Installing and using the dotnet-gcdump command-line tool.
+ms.date: 07/26/2020
+---
+# Heap analysis tool (`dotnet-gcdump`)
```

Comment:
Following the pattern of the other docs:
```suggestion
# dotnet-gcdump
```

Also it's best to avoid applying text styles to headings.

Reply:
The current heading is consistent with https://github.com/dotnet/docs/blob/master/docs/core/diagnostics/dotnet-dump.md
I wouldn't mind remove the text style from both the headings, but I personally like keeping the small description of the tool in the H1s. It also recommended that H1s be different from titles AFAIK.

Reply:
>keeping the small description of the tool in the H1s

OK

Reply:
```suggestion
# Heap analysis tool (dotnet-gcdump)
```